code/game/objects/structures/window.dm

```diff
@@ -21,6 +21,7 @@
 	var/junction = 0 //Because everything is terrible, I'm making this a window-level var
 	var/damageable = TRUE
 	var/deconstructable = TRUE
+	throwpass = FALSE
```

Comment:
no throwing things through windows?

Reply:
@hyper2snyper Throwpass does not actually have anything to do with throwing, it applies to projectiles.
...But windows have their own hit code which ignores this and just checks if it's a laser.
...and flamethrowers, which checks it to see if fire can go through it or not.

Yes it's dumb, I'm pretty sure it's been like that forever, maybe that used to share a var?

Reply:
you can just rename it in anothe r pr